tests/tracking/test_rest_tracking.py

```diff
@@ -2653,3 +2654,15 @@ def predict(self, context, model_input):
     )
     assert resp.status_code == 200
     assert model_info.model_id in resp.text
+
+
+def test_suppress_url_printing(mlflow_client: MlflowClient):
+    os.environ["MLFLOW_SUPPRESS_PRINTING_URL_TO_STDOUT"] = "true"
```

Comment:
```suggestion
def test_suppress_url_printing(mlflow_client: MlflowClient, monkeypatch):
    monkeypatch.setenv[MLFLOW_SUPPRESS_PRINTING_URL_TO_STDOUT.name] = "true"
```

You don't need to capture the output, you could simply test _log_url returns None :)

Reply:
Mind take another look? If I read it correctly _log_url always has no return. But updated to use monkeypatch